src/components/animated/_papercurl.less

```diff
@@ -0,0 +1,30 @@
+.papercurl-btn {
+  backface-visibility: hidden;
+  box-shadow: 0 0 1px rgba(0, 0, 0, 0);
+  display: inline-block;
+  position: relative;
+  transform: translateZ(0px);
+  vertical-align: middle;
+}
+.papercurl-btn:before {
```

Comment:
nest this in `.papercurl-btn` on line 1

Reply:
Can you please elaborate a little more. It would be of great help.
What exactly do you mean by nesting ?

Reply:
In Less you can nest rules for elements inside each other. For example you would add `.papercurl-btn:before` inside `.papercurl-btn` like this:

```
.papercurl-btn {
....
    &:before {
        ....
    }
}
```

where & refers to `.papercurl-btn`. You can read more about it in [the documentation for Less](http://lesscss.org/features/#parent-selectors-feature)

Reply:
Oh right I see,
Thank You so much.

Reply:
Hey since this PR got pretty messed up, would it be ok if I make a new one.

Reply:
Yes of course @wisetok12680  . I'll close this PR, you submit a new one.